src/components/Button.js

```diff
@@ -5,13 +5,16 @@ export const ButtonStyle = styled('button')`
   transition: ${props => props.theme.transition};
   color: ${props => props.theme.fg};
   ${props =>
-    props.buttonBorder &&
-    css`
-      border: 1px solid ${props => props.theme.fg};
-    `}
+    props.buttonBorder
+      ? css`
+          border: 5px solid ${props => props.theme.fg};
```

Comment:
Here, I would lean on the `css` operator as a return from a ternary statement if there are two or more things being affected. In `styled-components`, if the result of a expression is null, the entire CSS selector will be stricken from the record. 

```js
styled('div')`
  border: ${props => props.buttonBorder && `5px solid ${props.theme.fg}`}`
`
```

Reply:
I'm assuming (I might be wrong 🤔) that the second anon function call is not required, as props will already be in scope when I call `props.theme.fg`

### Something to think about (not required for this PR)

Also, if you can spend some time considering your naming convention for the theme object. Nick, imagine a new LHL grad was foisted into your role because you were injured. Would they understand the context of what `theme.fg` means? Is it a colour? Is it a width?

Proper naming would help with that. I tend to lean on the "bootstrap" naming conventions here. Names like `props.theme.color.success`, or `props.theme.color.secondary` help me, especially when I go back to it after a month without touching the thing.